### ISSUE TITLE
Domande / Punti di forza

### ISSUE THREAD

**Issue by leeoos** [airoprojects/visiope#5]
DOMANDE:
1. Quello che abbiamo fatto basta?
2. Metodi di data augmentation: aumentare il dataset in fase di preprocessing o aggiungere immagini trasformate in fase di traininig?
3. Vabene l'idea di aggiungere un generatore al posto de semplice uplampling? 
4. Va bene trainare con poche immagini causa poche risorse a disposizione?
5. Quali metriche sono piu interessanti da mostrare?
6.  Problemi nel training, in particolare nell'encoding della prediction e delle labels : https://www.jeremyjordan.me/semantic-segmentation/
7. Se le label non sono in one hot encoding la cross entropy viene falsata perche i valori vengono moltiplicati non per 1 ma per il valore della classe: H([1, 0], [0.8, 0.2]) = -(1 * (-0.097) + 0 * (-1.609))

PUNTI DI FORZA:
1. Dataloader fico
2. Possibilita di aumentare il dataset in preprocessing
3. Diverse loss da provare
4. Tre modelli ( stato dell'arte ) da testare + delle modifiche a segformer
